ochazuke/models.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Models and methods for working with the database."""
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects import postgresql
+
+db = SQLAlchemy()
+
+
+class Milestone(db.Model):
+    id = db.Column(postgresql.INTEGER, primary_key=True, unique=True)
+    title = db.Column(postgresql.TEXT, unique=True, nullable=False)
+    issues = db.relationship('Issue', backref='milestone', lazy=True)
+
+    def __repr__(self):
+        return '<Milestone {}: {}>'.format(self.id, self.title)
+
+
+issue_labels = db.Table('issue_labels',
+                        db.Column('issue_id',
+                                  postgresql.INTEGER,
+                                  db.ForeignKey('issue.id'),
+                                  primary_key=True),
+                        db.Column('label_id', postgresql.INTEGER,
+                                  db.ForeignKey('label.id'), primary_key=True)
+                        )
+
+
+class Label(db.Model):
```

Comment:
Which labels will we use in the dashboards?

Reply:
I figured we might want to be able to sort by browser, os, or type of issue at some point. I assumed we'd store all of them, but now that you mention it, we could also opt to only store certain ones. 🤔 

That might actually be better given that we're transitioning to a simpler labeling system. 💡

Reply:
yeah, always the balance in between fullproof and premature optimizations. It's usually slightly more delicate with DB because changing schemas are not fun. My hunch would be if we don't have plans for it right away, we should not put it at all. :) but if we have an idea of a dashboard with it. Let's do it. 

@adamopenweb @miketaylr opinions?

Reply:
If we don't want to parse all the labels now, we can do what Dennis suggested and store the labels in JSON in a single field. We can always parse as needed on the client side, which is what I'm currently doing.